sal_interfaces/MTM2/MTM2_Telemetry.xml

```diff
@@ -136,28 +184,35 @@
 <SALTelemetry>
     <Subsystem>MTM2</Subsystem>
     <EFDB_Topic>MTM2_temperaturesMeasured</EFDB_Topic>
-    <Description>Mirror and cell temperature sensor readings.</Description>
+    <Description>Mirror and cell temperature sensor readings relative to the origin (offset).</Description>
     <item>
         <EFDB_Name>ringTemperature</EFDB_Name>
         <Description>Ring temperatures: LG2-1, LG2-2, LG2-3, LG2-4, LG3-1, LG3-2, LG3-3, LG3-4, LG4-1, LG4-2, LG4-3, and LG4-4.</Description>
         <IDL_Type>double</IDL_Type>
-        <Units>Celsius</Units>
+        <Units>deg_C</Units>
         <Count>12</Count>
     </item>
     <item>
         <EFDB_Name>intakeTemperature</EFDB_Name>
         <Description>Intake temperatures: #1 and #2.</Description>
         <IDL_Type>double</IDL_Type>
-        <Units>Celsius</Units>
+        <Units>deg_C</Units>
         <Count>2</Count>
     </item>
     <item>
         <EFDB_Name>exhaustTemperature</EFDB_Name>
         <Description>Exhaust temperatures: #1 and #2.</Description>
         <IDL_Type>double</IDL_Type>
-        <Units>Celsius</Units>
+        <Units>deg_C</Units>
         <Count>2</Count>
     </item>
+    <item>
+        <EFDB_Name>origin</EFDB_Name>
+        <Description>Origin (offset).</Description>
```

Comment:
Shouldn't this be in an event, which gets published after we set the offset using the command?

Reply:
I also considered to use the event in the initial beginning. However, I felt it might be easier to do the calculation if we put the origin/ offset and temperature together if the analyzer does not know he/she should check the event to get the origin value. I can put it as an event if you like. Let us discuss this.

Reply:
Updated in commit: 7f353f2. Thanks!

Reply:
discussed with Te-Wei. We will make this an event.